### DISCUSSION TITLE
I created a project on C, but system still cannot find the path

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2569] [Q&A]
![image](https://user-images.githubusercontent.com/60806448/170912616-b62484fc-ed2a-4e99-b17c-5edd63aed530.png)
project created on C

![image](https://user-images.githubusercontent.com/60806448/170912663-a1bba8cc-a062-4923-a46e-89287a209f98.png)
still get the error

so um.. i guess it has nothing to do with me creating project on D

**Comment by RichardBray** [comment]
What command are you running to get this error?

Maybe there's an issue with the path in the Project.xml file 🤷
